### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by HanRu1** [WltyBY/Adaptive-Region-Specific-Loss#1]
How can I use it in my model training, can you give me an example

**Comment by WltyBY** [comment]
Just use it like a normal loss (CE loss or Dice Loss). There is a calculation example in the main function at the end of the code. Although this is not directly used in model training, I think it is enough to understand the use of this loss function. If you want a direct example, I think the Trainer in nnUNetv2 can help you.

**Comment by HanRu1** [comment]
thanks for your patient!

During my training, I noticed that the adaptive Tversky loss remains relatively high, while the cross-entropy loss and dice loss converge quickly. However, performance on the validation set improves faster. Is this behavior normal?

**Comment by WltyBY** [comment]
A few weeks ago, I changed the spatial summation of losses in the code to averaging. The value of the loss should be comparable to the common loss value, I think. Summing and averaging can affect the parameter update in terms of gradient magnitude. So the model converge quickly.

**Comment by junhanchen912** [comment]
您好，我最近也看到了这篇文章，看到您分享的代码非常感激，我有一些问题还不是很明白，可以向您请教一下吗？

**Comment by WltyBY** [comment]
@junhanchen912 当然。虽然我不一定能解答你的问题，但是请问。
